Tests/pens/ttGlyphPen_test.py

```diff
@@ -611,7 +613,7 @@ def test_cubic(self):
 
         for pen in (spen, ppen):
 
-            glyph = pen.glyph()
+            glyph = pen.glyph(preserveTopology=False)
```

Comment:
this seems no-op in this particular case. I would rather have distinct tests for this parameter with both True/False and expected outputs of each.

Reply:
I just added a new test. Will modify it to test True/False.